File: 119 Pascal's Triangle II/solution.py
class Solution:
    # @param {integer} rowIndex
    # @return {integer[]}
    def getRow(self, rowIndex):
        result = [1]
        for i in range(rowIndex):
            tmp = result
            result = [1]
            for j in range(i):
                result.append(tmp[j]+tmp[j+1])
            result.append(1)
        return result

File: 119 Pascal's Triangle II/solution.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<int> result;
        result.push_back(1);
        for (int i=0;i<rowIndex;i++){
            vector<int> tmp = result;
            for (int j=1;j<=i;j++){
                result[j] = tmp[j-1]+tmp[j];
            }
            result.push_back(1);
        }
        return result;
    }
};